sdk/monitor/azure-monitor-opentelemetry/azure/monitor/opentelemetry/_diagnostics/diagnostic_logging.py

```diff
@@ -69,6 +78,11 @@ def _initialize(cls):
                     _logger.addHandler(f_handler)
                     AzureDiagnosticLogging._initialized = True
 
+    @classmethod
+    def debug(cls, message: str, message_id: str):
```

Comment:
We should probably set logging level of `_logger` to be DEBUG if we are adding a `debug` method.

Reply:
Done.